src/main/java/at/hannibal2/skyhanni/config/features/misc/pets/PetConfig.java

```diff
@@ -28,4 +28,10 @@ public class PetConfig {
     @ConfigEditorBoolean
     @FeatureToggle
     public boolean hideAutopet = false;
+
+    @Expose
+    @ConfigOption(name = "Show Tier Boost", desc = "Shows an §c● §7by a pet when an tier boost core is equipped")
```

Comment:
Maybe it's just me but "an" sounds funny I think both could just be "a". Also missing the fullstop at the end of the description